pxd_fastpath_stub.c

```diff
@@ -24,4 +23,28 @@ int pxd_init_fastpath_target(struct pxd_device *pxd_dev, struct pxd_update_path_
 	return 0; // cannot fail
 }
 
+void pxd_fastpath_adjust_limits(struct pxd_device *pxd_dev, struct request_queue *topque) {}
+int pxd_suspend_state(struct pxd_device *pxd_dev) {return 0;}
+
+void pxd_suspend_io(struct pxd_device* pxd_dev) { }
+void pxd_resume_io(struct pxd_device* pxd_dev) { }
+int pxd_switch_fastpath(struct pxd_device* pxd_dev) {return -1;}
+int pxd_switch_nativepath(struct pxd_device* pxd_dev) {return -1;}
+
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,9,0)
+#define BLK_QC_RETVAL BLK_QC_T_NONE
+blk_qc_t pxd_make_request_fastpath(struct bio *bio)
+{
```

Comment:
This function should definitely assert hard. When fastpath function is stubbed, IO cannot be received/processed.